src/garage/tf/samplers/off_policy_vectorized_sampler.py

```diff
@@ -23,18 +24,27 @@ class OffPolicyVectorizedSampler(BatchSampler):
     """This class implements OffPolicyVectorizedSampler.
 
     Args:
-        algo(garage.np.RLAlgorithm): Algorithm.
-        env(garage.envs.GarageEnv): Environment.
-        n_envs(int): Number of parallel environments managed by sampler.
+        algo (garage.np.RLAlgorithm): Algorithm.
+        env (garage.envs.GarageEnv): Environment.
+        n_envs (int): Number of parallel environments managed by sampler.
+        no_reset (bool): Reset environment between samples or not.
+        scale_outputs (bool): Scale output by action space upper bound or
```

Comment:
i don't think this option should exist because, for instance, a gaussian policy cannot be meaningfully scaled. It is not the role of the sampler to modify the output of a policy.

Reply:
We probably need an environment wrapper in this case?

Reply:
How about make the algorithm responsible to scale?

Reply:
I don't think the algorithm should scale the actions either. Either the policy or the environment should do it.

Reply:
How about we do it in `normalized_env` as an option?

Reply:
I think an environment wrapper makes the most sense. What is the application for this?

Reply:
I thought `normalized_env` scaled actions. [see here](https://github.com/rlworkgroup/garage/blob/master/src/garage/envs/normalized_env.py#L95)

Reply:
Yes I just found out `normalized_env` already did that. I guess then we don't have to do any scaling anywhere else but we have to document clearly `normalized_env` is suggested to be used but also not to use if not appropriate, e.g. Gaussian policy on a Box env.

I will go ahead and remove the scaling.